app/src/main/java/com/infomaniak/mail/ui/login/LoginFragment.kt

```diff
@@ -181,6 +188,7 @@ class LoginFragment : Fragment() {
 
         animateColorChange(oldSecondaryBackground, newSecondaryBackground) { color ->
             requireActivity().window.statusBarColor = color
```

Comment:
Should you not remove the `statusBarColor` right above? It's not used anymore now right?